src/verify/VerificationStartScreen.tsx

```diff
@@ -143,25 +137,6 @@ function VerificationStartScreen({
     }
   }, [])
 
-  const canUsePhoneNumber = () => {
-    const countryCallingCode = country?.countryCallingCode || ''
-    if (
-      cachedNumber === phoneNumberInfo.e164Number &&
-      cachedCountryCallingCode === countryCallingCode
-    ) {
-      return true
-    }
-
-    const { SANCTIONED_COUNTRY } = getCountryFeatures(phoneNumberInfo.countryCodeAlpha2)
-    if (SANCTIONED_COUNTRY) {
-      dispatch(showError(ErrorMessages.COUNTRY_NOT_AVAILABLE))
-      return false
-    }
-
-    dispatch(setPhoneNumber(phoneNumberInfo.e164Number, countryCallingCode))
```

Comment:
this line shouldn't be dispatched until we've verified the number - this is still in the decentralised flow but it causes unverified numbers to be displayed in the app (like on the drawer menu)